### PR TITLE
Create todo list

### PR DESCRIPTION
**Pull request by jendiamond**
### Create Todo class methods to access tracks and slugs

+ slugs are the exercises that are written in each language
+ track are the languages

#### Methods:
  
**get_slugs**

    gets a list of all the unique exercises that currently exist in all the language tracks

**get_tracks**

    gets a list of all the language tracks

**get_existing_slugs(track)**

    gets a list of all the exercises that currently exist in a particular language track      

**get_todo(track)**

    gets a list of all the exercises that need to be written for a language track to have 
      all of the exercises that currently exist between all the tracks

==========================================

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.1.2'
+ruby '2.1.4'
```
Comment (kytrinyx) [inline Gemfile]: Please don't change the ruby version in the pull request (you can use `git add -p` to explicitly add individual patches to the staged index before commiting).

**Comment by kytrinyx** [comment]
Let's work together on Tuesday at the meetup to get this implemented as part of these two endpoints:

[`GET /problems`](http://x.exercism.io/problems), implemented in lib/app/routes/problems.rb on line 32 (the test is in test/app/routes/problems_test.rb on line 60)

[`GET /tracks`](http://x.exercism.io/tracks), implemented in lib/app/routes/tracks.rb on line 4 (the test is in test/app/routes/tracks_test.rb on line 13).

I don't want to have any implementation files at the root of the repository, I had you put it there last week since we were just spiking out the behavior.

**Comment by KalisaFalzone** [comment]
I see what you are saying, we can't just put everything in the root of the repository. Looking forward to working with you to get everything located in the appropriate places. See you tomorrow. :)

**Comment by kytrinyx** [comment]
Sweet! Yepp, see you tomorrow.

**Comment by kytrinyx** [comment]
I'm going to close this since the other PR was pulled in. Thank you :heart:

**Comment by KalisaFalzone** [comment]
Sounds good :)

**Pull request closed without merging**